lib/helpers/tsconfig.json

```diff
@@ -0,0 +1,24 @@
hulz (Charles M. Schulz's son and grandson, respectively), this movie is as heartwarming as it is innovative and interesting - a perfect fit for our Royal kids matinees!Our favourite underdog Charlie Brown has a crush on his new neighbour, the Little Red-Haired Girl, while his pup has fallen into the depths of his fantasy world as a World War I flying ace, pursuing his nemesis the Red Baron, all while trying to win the interests of a beautiful poodle named Fifi. Charlie Brown takes Lucy's advice and embarks on a series of new activities to gain the interest of the Little Red-Haired Girl, but with his ever present lack of confidence, has trouble making an impression on her, but he keeps trying anyway (we've all been there, Charlie Brown). This adorable flick about young puppy love is a surefire way to thaw out from the January weather, and with state of the art 3-D animation, it is undoubtedly the Peanuts like you've never seen them before! ","url":"https://www.universe.com/events/the-peanuts-movie-tickets-toronto-V8RJNB?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:20:36Z","startTBD":false,"endDateTime":"2016-01-17T21:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:20:36Z","startApproximate":false,"endDateTime":"2016-01-17T21:00:00Z","endApproximate":true},"s
```

Comment:
My git mistake. It's useless additional temporary file.

2016-07-28 15:44 GMT+03:00 Adam Meghji <notifications@github.com>:

> In lib/helpers/tsconfig.json
> <https://github.com/ticketmaster-api/sdk-javascript/pull/9#discussion_r72614404>
> :
>
> > +    "next":{
> > +      "href":"/discovery/v2/events/?view=null&page=1&size=20{&sort}",
> > +      "templated":true
> > +    }
> > +  },
> > +  "_embedded":{
> > +    "events":[
> > +      {
> > +        "name":"No Longer on Sale for Web",
> > +        "type":"event",
> > +        "id":"Z1lMVSyiJynZ177dJa",
> > +        "test":false,
> > +        "description":"Template for QA Event\r\n - Single GA Seating\r\n - Single Purchasing Price Type\r\n    - 1000 seat limit per visitor\r\n - 5000 Seats",
> > +        "url":"https://qapurchasetest.nbp.frontgatetickets.com/event/lwln7sy8bni2h448",
> > +        "locale":"en-us",
> > +        "images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true},{"ratio":"3_2","url":"https://static-label.frontgatetickets.com/common/events/92970.jpg","width":120,"height":80,"fallback":false}],"sales":{"public":{"startDateTime":"2014-01-01T20:15:00Z","startTBD":false,"endDateTime":"2020-04-02T01:15:00Z"}},"dates":{"start":{"localDate":"2014-03-31","localTime":"19:15:00","dateTime":"2014-03-31T19:15:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"dateTime":"2020-04-01T21:15:00Z","approximate":false},"timezone":"America/Chicago","status":{"code":"onsale"}},"_links":{"self":{"href":"/discovery/v2/events/Z1lMVSyiJynZ177dJa?locale=en-us"},"venues":[{"href":"/discovery/v2/venues/Z1lMVSyiJynZaec-vPad1Z?locale=en-us"}]},"_embedded":{"venues":[{"name":"FGS - Selenium (With enough text for 2nd","type":"venue","id":"Z1lMVSyiJynZaec-vPad1Z","test":false,"locale":"en-us","postalCode":"78704","city":{"name":"Austin"},"state":{"stateCode":"TX"},"country":{"name":"United States Of America","countryCode":"US"},"address":{"line1":"1711 S. Congress","line2":"2nd Floor"},"location":{"longitude":"-97.7658","latitude":"30.2433"},"_links":{"self":{"href":"/discovery/v2/venues/Z1lMVSyiJynZaec-vPad1Z?locale=en-us"}}}]}},{"name":"OSEA Membership Registration","type":"event","id":"LvZ184j83k1A5TPn6ZZgc","test":false,"description":"If you need any assistance in choosing your membership category, please do not hesitate to contact us at 416-977-4441.Your annual membership fee is an investment that enables OSEA to research and champion consistent and supportive political and regulatory environment necessary for Community Power proponents and the entire sustainable energy sector to thrive. For over 10 years, OSEA has been advocating progressive and inclusive energy policies that are based on international best practices. Successes to date include:\nThe creation of the feed-in tariff programs\nThe campaign for the Green Energy and Green Economy Act\nThe creation of the Community, Aboriginal and Municipal Energy Partnership programs\nThe aboriginal loan guarantee program\nCreation of the Community Power Fund\nThe Renewable Energy Standard Offer Program\nOSEA continues to consult with its members on issues of importance and research best practices to inform its regulatory work and advice to the government on how best to support the Community Power and larger sustainable energy sector.In addition to researching and advocating for best practices at the policy and regulatory level, OSEA also offers its members training, advisory and facilitation services. OSEA also holds an annual Community Power conference and other networking opportunities providing suppliers, manufacturers service providers and developer’s opportunities to collaborate and build their businesses.Membership is open to any individual or organization that wishes to support OSEA’s values, vision and mission upon approval of the membership committee.All current members are eligible to receive a discount of 15% if they renew 30 days prior to their renewal date. Membership renewal fees are due 12 months following application approval and initial fee payment. All membership applications are subject to approval of the membership committee.For any issues with payment or to renew your membership, please contact  the OSEA Office Manager, at admin(at)ontario-sea.org or by phone at 416-977-4441 ext: 1.","url":"https://www.universe.com/events/osea-membership-registration-tickets-toronto-1PDMTJ?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-29T21:39:21Z","startTBD":false,"endDateTime":"2016-08-01T04:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-29T21:39:21Z","startApproximate":false,"endDateTime":"2016-08-01T04:00:00Z","endApproximate":true},"start":{"localDate":"2015-08-06","localTime":"22:15:00","dateTime":"2015-08-07T02:15:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"00:00:00","dateTime":"2016-08-01T04:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"onsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184j83k1A5TPn6ZZgc?locale=en-us"}}},{"name":"Football: 2016 Season Pass","type":"event","id":"LvZ184JJ8Apl_vc1eZvT3","test":false,"description":"2016 Guelph Gryphon Football Season tickets are now available! Come watch your Guelph Gryphons as they defend their Yates Cup title!Ticket Prices:General Admission Season Pass: $80.002016 Home Schedule:August 28th: Toronto Varsity Blues @ Guelph 1PMSeptember 10th: Queens Gaels @ Guelph 1PMSeptember 24th: Western Mustangs @ Guelph 1PMOctober 22nd: York Lions @ Guelph 1PM*Please note that the purchase of these tickets is non-refundable.","url":"https://www.universe.com/events/football-2016-season-pass-tickets-24RQPX?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true},{"ratio":"1_1","url":"https://images.universe.com/5f629400-5d6c-413d-92a3-e6ff3fbc5558/-/resize/1200x/-/crop/1200x1200/0,0/-/format/jpeg/-/scale_crop/500x500/center/-/progressive/yes/-/inline/yes/","width":500,"height":500,"fallback":false}],"place":{"city":{"name":"Guelph"},"country":{"name":"Canada","countryCode":"CA"},"address":{"line1":"Alumni Stadium","line2":"26 Powerhouse Ln, Guelph, ON N1G 3B9, Canada"},"location":{"longitude":"-80.22722999999996","latitude":"43.53469"},"area":{},"state":{"name":"Ontario","stateCode":"ON"},"postalCode":"N1G 3B9"},"sales":{"public":{"startDateTime":"2016-01-01T00:00:00Z","startTBD":false,"endDateTime":"2016-08-28T17:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-01T00:00:00Z","startApproximate":false,"endDateTime":"2016-08-28T17:00:00Z","endApproximate":false},"start":{"localDate":"2015-12-10","localTime":"11:00:00","dateTime":"2015-12-10T16:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"11:00:00","dateTime":"2016-08-28T17:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"onsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184JJ8Apl_vc1eZvT3?locale=en-us"}}},{"name":"Savoury & Sweet Bowen - a food-tasting tour","type":"event","id":"LvZ184NvUVlYs-gEsZ7GT","test":false,"description":"Enjoy a delectable Bowen Island experience. Here's your chance to savour an array of culinary delights selected from a variety of restaurants, food and beverage purveyors. Savoury &amp; Sweet is a food-tasting tour that reflects the island's diverse cultural influences and casual, laid-back style - from hip Asian-influenced rice dishes to fine French bistro fare, with lots of handmade, organic samplings in between. Also on the menu, a \"taste\" of Bowen's rich history and highlights, including a visit to Davies Heritage Orchard. Join us for this fun and delicious introduction to Bowen Island. This food tour reflects seasonal offerings, so venues and menus are subject to change. The tour includes stops at 8 or more venues with 10+ samplings. So, please bring your appetite along!Our partners include:\nRustique Bistro - West-Coast influenced French foods\nLime &amp; Moon Pie Company - handmade organic pies\nShika Provisions - Asian fusion \nThe Snug Cafe - a Bowen institution\nThe Ruddy Potato Deli\nDoc Morgan's\nBowen Island Coffee\nBowen Island Beer &amp; Wine Store\nCandy in the Cove - self-proclaimed world's smallest candy store\nJam Works - locally sourced and made preserves\nCocoa West Chocolatier - handmade organic chocolates\nDavies Heritage Orchard\nThings you should know about this tour:\nPlease wear comfortable walking shoes, as you will be walking on mixed and uneven surfaces.\nTours run rain or shine, so please wear appropriate clothing.\nTours require a minimum number of participants. Your ticket will be refunded or transferred to another date if the minimum number of participants for a specific tour is not met. We will give you 48-hour notice of any tour cancellations.\nWe understand that sometimes plans change. We will endeavor to accommodate changes to your tour booking, but refunds or tour rescheduling requires a minimum of 3 days notice. Please contact Jody at Bowen Island Tours directly via, email, phone or text, to discuss any cancellations or rescheduling.\nWe are unable to provide refunds or reschedule tours if you arrive late. Sometimes the ferry runs late and there are vehicle overloads. So please ensure that you allow enough time for these circumstances.\nAll tour participants are required to read and accept Bowen Island Tours' Waiver and Release of Liability before purchasing tickets. You will be asked to accept the terms at the ticket checkout. Please read the terms on Bowen Island Tours' website located on the home page, bottom footer: www.bowenislandtours.com\nPlease contact Jody at Bowen Island Tours if you have questions about this or any other tours.\n* If you do not see a day and time that fits with your schedule, please contact us. Often our schedule changes and we can accommodate requests or advise of upcoming dates.","url":"http://localhost:3000/events/savoury-sweet-bowen-a-food-tasting-tour-tickets-bowen-island-61TVG?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2015-12-10T19:12:58Z","startTBD":false,"endDateTime":"2015-12-30T22:00:00Z"}},"dates":{"access":{"startDateTime":"2015-12-10T19:12:58Z","startApproximate":false,"endDateTime":"2015-12-30T22:00:00Z","endApproximate":true},"start":{"localDate":"2015-12-30","localTime":"11:30:00","dateTime":"2015-12-30T19:30:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"14:00:00","dateTime":"2015-12-30T22:00:00Z","approximate":false},"timezone":"America/Vancouver","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184NvUVlYs-gEsZ7GT?locale=en-us"}}},{"name":"FORGE: Arts & Business Management Workshop","type":"event","id":"LvZ184JwGmL6zhtCsZZq6","test":false,"description":"FORGE: Arts &amp; Business ManagementJanuary 14, 6:30- 9:00pm.Committee Room 2, Toronto City Hall- 100 Queen Street WestIn this workshop, Ben Benedict (Benedict Creative Communications) will provide guidance on art career management and operating a small business. Topics for discussion will include effective marketing, sourcing private and public funding, exhibition opportunities, proper artist fee payments, effective tax and bookkeeping methods and professionally documenting your artwork and accomplishments. Following the presentation, there will be a Q&amp;A and group discussion.The session will be divided into the following sections:Artist Marketing and Publicity (Print, Social media, online, networking)Exhibition Opportunities (OAAG, Artist Run Centres, Venues, Commercial Galleries)Getting Paid/Funding  (CARFAC Fees, Copyright, Grant applications)Business Structure &amp; Bookkeeping (Income Tax, Federal/Provincial Sales Tax, Insurance, Bookkeeping)Curriculum Vitae (Documenting your artwork and career).Tickets: $10 (To cover presenter fee)Bio: Ben Benedict of Benedict Creative Communications is recognized as one of Ontario’s leading culture workers as a writer/editor; visual artists/curator; public relations and corporate communications professional and educator; and a sought out public speaker on The Business of the Arts, with the most recent presentation being Marketing 2.0 as part of a CARFAC: Webinar Series for Visual Artists across Canada in October 2014","url":"https://www.universe.com/events/forge-arts-business-management-workshop-tickets-toronto-HVSJ74?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T23:04:53Z","startTBD":false,"endDateTime":"2016-01-15T02:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T23:04:53Z","startApproximate":false,"endDateTime":"2016-01-15T02:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-14","localTime":"18:30:00","dateTime":"2016-01-14T23:30:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"21:00:00","dateTime":"2016-01-15T02:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184JwGmL6zhtCsZZq6?locale=en-us"}}},{"name":"MILF Life Crisis- a new comedy about dating again!","type":"event","id":"LvZ184J08DaVajtl-ZvmO","test":false,"description":"What happens when you're newly divorced with kids? You go dating every other weekend! A new kind of relationship comedy! Because after marriage comes dating! Written &amp; Performed by Anne Marie Scheffler of Second City Toronto, Just For Laughs, The Comedy Network, SPANK! The Fifty Shades Parody, and Carolines on BroadwaySo you dated badly in your twenties and thirties, and just in the nick of time, you snagged a soulmate and started a family. But now the soulmate doesn't love you anymore, you had to sell the house to pay for the divorce lawyers, and the kids are with you only fifty percent of the time. Did you really just land back on the dating market? What kind of cruel world is this? Do you really need to go through first dates and self judgement all over again? Or is this happening to everybody: sexy confident adults who can have amazing relationships and not be trying to get married and start a family. Cougar town? Utopia? Or just a lot of wine and lingerie? Who are these guys you're dating? A new kind of fun!","url":"https://www.universe.com/events/milf-life-crisis-a-new-comedy-about-dating-again-tickets-toronto-1D7FTB?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:38:05Z","startTBD":false,"endDateTime":"2016-01-15T01:15:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:38:05Z","startApproximate":false,"endDateTime":"2016-01-15T01:15:00Z","endApproximate":true},"start":{"localDate":"2016-01-14","localTime":"19:00:00","dateTime":"2016-01-15T00:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"20:15:00","dateTime":"2016-01-15T01:15:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184J08DaVajtl-ZvmO?locale=en-us"}}},{"name":"Neon Dreams Cinema Club presents: MIAMI BLUES","type":"event","id":"LvZ184Ikwnxy_phb1ZvA8","test":false,"description":"Welcome to a world where people's worst impulses and darkest desires are stimulated by a landscape of synth-pop and bright neon lights. Nothing is quite as it seems as you surrender yourself to the dazzling sights and soothing sounds of the Neon Dreams Cinema Club, a monthly film series exclusive to The Royal Cinema bringing you the best in delightfully surreal neo-noir cinema from the 70's, 80's, and beyond.Pre-show starts at 7:00pmFeature starts at 8:00pmVISIT THE EVENT PAGE&gt;&gt;HERE&lt;&lt;This 35mm presentation is made possible thanks to our lovely sponsor Sonic Boom and will be preceded by a mind-shattering pre-show, featuring a themed video mixtape of music, film, and television to put you in the right mood. As always this will be a lisenced event, courtesy of Monarch Tavern, and there will be AMAZING PRIZES given out to many lucky patrons. Be sure to arrive early!======After Junior (Alec Baldwin) is released from prison, he plans on starting a new life in Miami. But when he kills a man in the airport, he flees the scene and finds Susie (Jennifer Jason Leigh), a mild-mannered prostitute searching for stability. The two opposites become romantically involved, and Junior steals a badge and gun from a veteran detective (Fred Ward). Using the officer's identity, Junior embarks on a crime spree and convinces Susie that he is the perfect man.\"George Armitage's Miami Blues is another film slightly ahead of its time. In 1990 its combination of tough, funny noir and flashes of shocking violence seemed to bemuse audiences. Only five years later would Tarantino turn what was once for cult appreciation only into a global smasheroo. This very entertaining Miami lowlife romp has at its heart two great badass performances, the grizzled Fred Ward as Hoke Mosely and the frequently shirtless and impressively hairy Alec Baldwin as the charming psychopath Junior. This film is a little gem of broken fingers, lost digits and missing teeth. Anyone who has seen my amateur film Dead Right will know that I stole a joke from Miami Blues.\"- Edgar Wright (director of Shaun Of The Dead, Scott Pilgrim Vs. The World)","url":"https://www.universe.com/events/neon-dreams-cinema-club-presents-miami-blues-tickets-toronto-TFM67Q?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T21:36:14Z","startTBD":false,"endDateTime":"2016-01-15T02:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T21:36:14Z","startApproximate":false,"endDateTime":"2016-01-15T02:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-14","localTime":"19:00:00","dateTime":"2016-01-15T00:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"21:00:00","dateTime":"2016-01-15T02:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184Ikwnxy_phb1ZvA8?locale=en-us"}}},{"name":"Suffragette","type":"event","id":"LvZ184IkRydGe5f3hZvey","test":false,"description":"A drama that tracks the story of the foot soldiers of the early feminist movement, women who were forced underground to pursue a dangerous game of cat and mouse with an increasingly brutal State.These women were not primarily from the genteel educated classes, they were working women who had seen peaceful protest achieve nothing. Radicalized and turning to violence as the only route to change, they were willing to lose everything in their fight for equality - their jobs, their homes, their children and their lives.Maud was one such foot soldier. The story of her fight for dignity is as gripping and visceral as any thriller, it is also heart-breaking and inspirational.-Alamo Drafthouse","url":"https://www.universe.com/events/suffragette-tickets-toronto-MQ0WV1?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:31:20Z","startTBD":false,"endDateTime":"2016-01-16T02:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:31:20Z","startApproximate":false,"endDateTime":"2016-01-16T02:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-15","localTime":"19:00:00","dateTime":"2016-01-16T00:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"21:00:00","dateTime":"2016-01-16T02:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184IkRydGe5f3hZvey?locale=en-us"}}},{"name":"MILF Life Crisis- a new comedy about dating again!","type":"event","id":"LvZ184J08DaVajtl-Zvm-","test":false,"description":"What happens when you're newly divorced with kids? You go dating every other weekend! A new kind of relationship comedy! Because after marriage comes dating! Written &amp; Performed by Anne Marie Scheffler of Second City Toronto, Just For Laughs, The Comedy Network, SPANK! The Fifty Shades Parody, and Carolines on BroadwaySo you dated badly in your twenties and thirties, and just in the nick of time, you snagged a soulmate and started a family. But now the soulmate doesn't love you anymore, you had to sell the house to pay for the divorce lawyers, and the kids are with you only fifty percent of the time. Did you really just land back on the dating market? What kind of cruel world is this? Do you really need to go through first dates and self judgement all over again? Or is this happening to everybody: sexy confident adults who can have amazing relationships and not be trying to get married and start a family. Cougar town? Utopia? Or just a lot of wine and lingerie? Who are these guys you're dating? A new kind of fun!","url":"https://www.universe.com/events/milf-life-crisis-a-new-comedy-about-dating-again-tickets-toronto-1D7FTB?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:39:23Z","startTBD":false,"endDateTime":"2016-01-16T01:15:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:39:23Z","startApproximate":false,"endDateTime":"2016-01-16T01:15:00Z","endApproximate":true},"start":{"localDate":"2016-01-15","localTime":"19:00:00","dateTime":"2016-01-16T00:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"20:15:00","dateTime":"2016-01-16T01:15:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184J08DaVajtl-Zvm-?locale=en-us"}}},{"name":"Desi Night","type":"event","id":"LvZ184t4Qfovi_GmuZZRd","test":false,"description":"January 15DESI NIGHTTIME:  DOORS 9PM \n*THIS IS A CLOSED CAMPUS EXCLUSIVE SSU EVENT* *MUST BE STUDENT OF SHERIDAN COLLEGE OR GUEST OF STUDENT TO ATTEND**GUEST MUST BE 19+ AND HAVE A SEPARATE TICKET*","url":"https://www.universe.com/events/desi-night-tickets-brampton-F06SBV?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-16T03:57:26Z","startTBD":false,"endDateTime":"2016-01-16T06:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-16T03:57:26Z","startApproximate":false,"endDateTime":"2016-01-16T06:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-15","localTime":"21:00:00","dateTime":"2016-01-16T02:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"01:00:00","dateTime":"2016-01-16T06:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184t4Qfovi_GmuZZRd?locale=en-us"}}},{"name":"Kensington Krawl Food & Culture Tour","type":"event","id":"LvZ184Ii6JrzSXDEuZZcN","test":false,"description":"Explore the diverse tastes and flavours of Toronto's most eclectic neighbourhoodKensington Market emerged during the early 1900’s, when Jewish immigrants settled in the neighbourhood. Many of the residents began selling their goods and wares from pushcarts and horse-drawn carriages. Within the boundary of a few square blocks located west of Spadina, a bustling area with narrow streets of residential complexes comprising of small stores and colourful awnings was formed to create an open-air market. Throughout the decades, the market gradually evolved and transformed as each new successive waves of immigrants – European Jews, Italians, Portuguese, Chinese, Jamaican – left their mark on the community and added another layer of cultural diversity.For our “Kensington Krawl” food tour, you will have the opportunity to enjoy select tastings in the market including from a cheese shop, bakery, delicatessen, butcher shop, spice shop, and more. You will meet with local merchants and shopkeepers whenever possible during your excursion. Your guide will offer you a background on the history and heritage of this vibrant neighbourhood, with memorable stories and anecdotes. You will discover why Kensington Market is truly a one-of-a-kind place!\nStart Time: 11:30 am.\nDuration: Approximately 3.5 hours.\nCapacity: Up to 12 guests.\nPhysical Demands: Walking at a leisurely pace - total distance is approx. 1.3 km, standing room only for most stops.\nWeather Impact: Tour takes place rain or shine (tour will be adjusted to the weather).\nFood: 6 generous tastings are done within Kensington Market including from a cheese shop, delicatessen, butcher shop, spice shop, sweet shop, and other food establishments.\nDietary Substitutions: Vegetarian and a non-alcoholic option are available. Unfortunately at this time no substitutions can be offered for any other food allergies or restrictions such as gluten-free or dairy-free. We sincerely appreciate your understanding in this matter. ","url":"https://www.universe.com/events/kensington-krawl-food-culture-tour-tickets-toronto-GZ6RV?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:38:05Z","startTBD":false,"endDateTime":"2016-01-16T20:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:38:05Z","startApproximate":false,"endDateTime":"2016-01-16T20:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-16","localTime":"11:30:00","dateTime":"2016-01-16T16:30:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"15:00:00","dateTime":"2016-01-16T20:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184Ii6JrzSXDEuZZcN?locale=en-us"}}},{"name":"The Peanuts Movie","type":"event","id":"LvZ184IknB3y_p3npZvvT","test":false,"description":"The classic characters that we've known and loved for decades have returned in the first feature length film in 35 years! Written by Craig and Bryan Schulz (Charles M. Schulz's son and grandson, respectively), this movie is as heartwarming as it is innovative and interesting - a perfect fit for our Royal kids matinees!Our favourite underdog Charlie Brown has a crush on his new neighbour, the Little Red-Haired Girl, while his pup has fallen into the depths of his fantasy world as a World War I flying ace, pursuing his nemesis the Red Baron, all while trying to win the interests of a beautiful poodle named Fifi. Charlie Brown takes Lucy's advice and embarks on a series of new activities to gain the interest of the Little Red-Haired Girl, but with his ever present lack of confidence, has trouble making an impression on her, but he keeps trying anyway (we've all been there, Charlie Brown). This adorable flick about young puppy love is a surefire way to thaw out from the January weather, and with state of the art 3-D animation, it is undoubtedly the Peanuts like you've never seen them before! ","url":"https://www.universe.com/events/the-peanuts-movie-tickets-toronto-V8RJNB?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:17:29Z","startTBD":false,"endDateTime":"2016-01-16T21:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:17:29Z","startApproximate":false,"endDateTime":"2016-01-16T21:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-16","localTime":"14:00:00","dateTime":"2016-01-16T19:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"16:00:00","dateTime":"2016-01-16T21:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184IknB3y_p3npZvvT?locale=en-us"}}},{"name":"Brewers' Dinner at Folly Brewpub","type":"event","id":"LvZ184IJeHyYdBZh6ZdjF","test":false,"description":"Join us for our first Brewers' Dinner on January 16.Featuring *4-course tasting menu, paired with beers selected by our brewers, Chris &amp; Christina. *The opportunity to taste a single-release beer, available only to ticket holders*An evening where you have the opportunity to interact with our brewers, explore the brewery, and learn about what's next at Folly A limited amount of tickets are available for this event, so be sure to grab yours now!  ","url":"https://www.universe.com/events/brewers-dinner-at-folly-brewpub-tickets-toronto-VJQC5K?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-16T23:19:19Z","startTBD":false,"endDateTime":"2016-01-17T02:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-16T23:19:19Z","startApproximate":false,"endDateTime":"2016-01-17T02:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-16","localTime":"18:00:00","dateTime":"2016-01-16T23:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"21:00:00","dateTime":"2016-01-17T02:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184IJeHyYdBZh6ZdjF?locale=en-us"}}},{"name":"MILF Life Crisis- a new comedy about dating again!","type":"event","id":"LvZ184J08DaVajtl-Zvmx","test":false,"description":"What happens when you're newly divorced with kids? You go dating every other weekend! A new kind of relationship comedy! Because after marriage comes dating! Written &amp; Performed by Anne Marie Scheffler of Second City Toronto, Just For Laughs, The Comedy Network, SPANK! The Fifty Shades Parody, and Carolines on BroadwaySo you dated badly in your twenties and thirties, and just in the nick of time, you snagged a soulmate and started a family. But now the soulmate doesn't love you anymore, you had to sell the house to pay for the divorce lawyers, and the kids are with you only fifty percent of the time. Did you really just land back on the dating market? What kind of cruel world is this? Do you really need to go through first dates and self judgement all over again? Or is this happening to everybody: sexy confident adults who can have amazing relationships and not be trying to get married and start a family. Cougar town? Utopia? Or just a lot of wine and lingerie? Who are these guys you're dating? A new kind of fun!","url":"https://www.universe.com/events/milf-life-crisis-a-new-comedy-about-dating-again-tickets-toronto-1D7FTB?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:39:23Z","startTBD":false,"endDateTime":"2016-01-17T01:15:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:39:23Z","startApproximate":false,"endDateTime":"2016-01-17T01:15:00Z","endApproximate":true},"start":{"localDate":"2016-01-16","localTime":"19:00:00","dateTime":"2016-01-17T00:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"20:15:00","dateTime":"2016-01-17T01:15:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184J08DaVajtl-Zvmx?locale=en-us"}}},{"name":"MDFF presents: TIRED MOONLIGHT","type":"event","id":"LvZ184IYRAnMGdLeXZZ30","test":false,"description":"Combustible dreams fail to ignite as lonely, middle-aged, Dawn is confronted by lost love in a glorified-pit-stop town. Every town has a post office, lovers, guns, switchblades and beer. you just have to know where to look and when to look the other way. Pitting grand landscapes against dinners of fried chicken and the roar of V8 engines on Saturday nights, TIRED MOONLIGHT wanders through Solitaire games (always won), secrets lost in cavernous hearts, and the fifty miles of bad road that always gets you home.","url":"https://www.universe.com/events/mdff-presents-tired-moonlight-tickets-toronto-KGT9MY?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-17T01:00:52Z","startTBD":false,"endDateTime":"2016-01-17T03:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-17T01:00:52Z","startApproximate":false,"endDateTime":"2016-01-17T03:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-16","localTime":"20:00:00","dateTime":"2016-01-17T01:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"22:00:00","dateTime":"2016-01-17T03:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184IYRAnMGdLeXZZ30?locale=en-us"}}},{"name":"Folly Brewery Tour","type":"event","id":"LvZ184IJb3UbwBrRgZd_J","test":false,"description":"Intimate tours with our brewers, Chris &amp; Christina. Every Sunday afternoon. ","url":"https://www.universe.com/events/folly-brewery-tour-tickets-toronto-PX9LY4?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-17T18:57:04Z","startTBD":false,"endDateTime":"2016-01-17T19:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-17T18:57:04Z","startApproximate":false,"endDateTime":"2016-01-17T19:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-17","localTime":"13:00:00","dateTime":"2016-01-17T18:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"14:00:00","dateTime":"2016-01-17T19:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184IJb3UbwBrRgZd_J?locale=en-us"}}},{"name":"Folly Brewery Tour","type":"event","id":"LvZ184IJb3UbwBrRgZd_E","test":false,"description":"Intimate tours with our brewers, Chris &amp; Christina. Every Sunday afternoon. ","url":"https://www.universe.com/events/folly-brewery-tour-tickets-toronto-PX9LY4?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-17T18:57:20Z","startTBD":false,"endDateTime":"2016-01-17T20:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-17T18:57:20Z","startApproximate":false,"endDateTime":"2016-01-17T20:00:00Z","endApproximate":true},"start":{"localDate":"2016-01-17","localTime":"14:00:00","dateTime":"2016-01-17T19:00:00Z","dateTBD":false,"dateTBA":false,"timeTBA":false,"noSpecificTime":false},"end":{"localTime":"15:00:00","dateTime":"2016-01-17T20:00:00Z","approximate":false},"timezone":"America/Toronto","status":{"code":"offsale"}},"_links":{"self":{"href":"/discovery/v2/events/LvZ184IJb3UbwBrRgZd_E?locale=en-us"}}},{"name":"The Peanuts Movie","type":"event","id":"LvZ184IknB3y_p3npZveZ","test":false,"description":"The classic characters that we've known and loved for decades have returned in the first feature length film in 35 years! Written by Craig and Bryan Schulz (Charles M. Schulz's son and grandson, respectively), this movie is as heartwarming as it is innovative and interesting - a perfect fit for our Royal kids matinees!Our favourite underdog Charlie Brown has a crush on his new neighbour, the Little Red-Haired Girl, while his pup has fallen into the depths of his fantasy world as a World War I flying ace, pursuing his nemesis the Red Baron, all while trying to win the interests of a beautiful poodle named Fifi. Charlie Brown takes Lucy's advice and embarks on a series of new activities to gain the interest of the Little Red-Haired Girl, but with his ever present lack of confidence, has trouble making an impression on her, but he keeps trying anyway (we've all been there, Charlie Brown). This adorable flick about young puppy love is a surefire way to thaw out from the January weather, and with state of the art 3-D animation, it is undoubtedly the Peanuts like you've never seen them before! ","url":"https://www.universe.com/events/the-peanuts-movie-tickets-toronto-V8RJNB?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:20:36Z","startTBD":false,"endDateTime":"20